src/Netatmo/Clients/NAWSApiClient.php

```diff
@@ -21,7 +21,14 @@ class NAWSApiClient extends NAApiClient
    */
    public function getData($device_id = NULL, $get_favorites = TRUE)
    {
-       return $this->api('getstationsdata', 'GET', array($device_id, $get_favorites));
+		    $params = array();
```

Comment:
Good catch!
Could you please fix the indentation on this line though?
```suggestion
        $params = array();
```